requirements.txt

```
@@ -1,4 +1,4 @@
 python-dxf==7.8.0
 tqdm
 typer
-pydantic
+pydantic>=1.5,<3,!=2.0.*
```

```suggestion
pydantic>=1.5,<3
```
The part `!=2.0.*` doesn't come from docker-charon, it comes from python-on-whales. It's the responsibility of python-on-whales to impose this restriction.

---

Ok I didn't notice the restriction was about python-on-whales only, done !